src/python-bindings/condor_python.h

```diff
@@ -35,13 +35,9 @@
 
 #if defined(_CONDOR_COMMON_FIRST)
     #include "condor_common.h"
-    // This #define confuses linking on Windows.
-    // (It links the non-minor-version-specific
-    // python3.lib, which doesn't exist.)
-    #define Py_LIMITED_API
 #endif /* _CONDOR_COMMON_FIRST */
 
-
+#define Py_LIMITED_API 3
```

Comment:
Can we get a comment here about why "3", when we might want to change it, and what other things (like .abi3.so) that might need to change at the same time?

Reply:
That's a good idea.